test/integration/freetext_editor_spec.js

```diff
@@ -837,4 +839,219 @@ describe("Editor", () => {
       );
     });
   });
+
+  describe("FreeText (move existing)", () => {
+    let pages;
+
+    beforeAll(async () => {
+      pages = await loadAndWait("freetexts.pdf", ".annotationEditorLayer");
+    });
+
+    afterAll(async () => {
+      await closePages(pages);
+    });
+
+    it("must move an annotation", async () => {
+      await Promise.all(
+        pages.map(async ([browserName, page]) => {
+          await page.click("#editorFreeText");
+
+          const editorIds = await getEditors(page, "freeText");
+          expect(editorIds.length).withContext(`In ${browserName}`).toEqual(6);
+
+          // All the current annotations should be serialized as null objects
+          // because they haven't been edited yet.
+          const serialized = await getSerialized(page);
+          expect(serialized).withContext(`In ${browserName}`).toEqual([]);
+
+          // DnD doesn't work well for the moment hence the test is disabled.
```

Comment:
Is there an existing (upstream) bug or issue that we can reference here, so that we can eventually re-enable this test at some future point?

Reply:
I finally managed to make it running with Chrome but not with Firefox (see https://bugzilla.mozilla.org/show_bug.cgi?id=1838638).